scss/reflex/_mixins.scss

```diff
@@ -104,13 +104,13 @@
     @include loop-grid--order-helpers($reflex-columns);
 }
 
-@mixin loop-grid--order-helpers($index) {
+@mixin loop-grid--order-helpers($index, $breakpoint: null) {
     @if $index >= 0 {
         .#{$reflex-prefix}grid--order-#{$index} {
```

Comment:
I think we need to output the `$breakpoint` here like so:

`.#{$reflex-prefix}grid--order-#{$index}#{$breakpoint} {`

Thanks for this PR, I am not going to merge it directly but am incorporating it into the library myself.